docs/content/contributing/internal/RemoteCache.md

```diff
@@ -1,46 +1,13 @@
-# Remote Cache
+# How to enable remote cache for local work
 
 ???+ warning 
-    Currently available only for Avito employees via corporate network
+    Available only for Avito employees via corporate network
 
-[Official docs](https://docs.gradle.com/build-cache-node)
-
-## How to enable for local work
-
-???+ Prerequisites 
-    Ask for `GRADLE_CACHE_NODE_HOST` in internal chat
-    
-Set environment variable `GRADLE_CACHE_NODE_HOST`
-
-## How to deploy
-
-???+ Prerequisites
-    - DOCKER_REGISTRY env set
-    - GRADLE_CACHE_NODE_HOST env set
-    - [kubernetes credentials configured](http://links.k.avito.ru/kubectl)
-    - Write access to `gradle-remote-cache` namespace in cluster (see MBS-5444)
-
-???+ warning
-    Persistence volume not set, deleting a pod will lead to losing cache.
-
-`make deploy_gradle_cache_node`
+Add gradle properties to your `$GRADLE_USER_HOME/gradle.properties`:
```

Comment:
`$GRADLE_USER_HOME` - to be precise, this env is optional

Reply:
Do you know a better way to express that it's not always a `~/.gradle`?

Reply:
[`GRADLE_USER_HOME`](https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_environment_variables)